src/Notice.tsx

```diff
@@ -49,6 +50,14 @@ const Notify = React.forwardRef<HTMLDivElement, NoticeProps & { times?: number }
     onNoticeClose(eventKey);
   };
 
+  const handleCloseKeydown: React.KeyboardEventHandler<HTMLAnchorElement> = closable
```

Comment:
`handleCloseKeydown` -> `onCloseKeyDown`

Reply:
no need `closable` check since close btn will not show when `closable` is `false`.